.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+
+<!-- Why was this PR created? -->
+
+## Development notes
+
+<!-- What have you changed? Consider adding a screenshot or GIF. -->
+
+## QA notes
+
+<!-- How has the expected behaviour changed? What testing strategies have you used? -->
```

Comment:
Might be worth adding a mention to involve designers in QA -
```suggestion
<!-- How has the expected behaviour changed? What testing strategies have you used? Have you QAed your final work with the design team? -->
```

Reply:
Nice. Will add.